### PR TITLE
Clippy on osm2lanes-web

### PR DESCRIPTION
**Pull request by droogmic**

### CODE REVIEW

File: osm2lanes-web/src/control.rs [hunk 1]
```diff
@@ -52,11 +52,11 @@ impl Component for Control {
                 let examples: BTreeMap<_, _> = tests
                     .into_iter()
                     .filter_map(|t| {
-                        let example_name = t.example().map(|e| e.to_owned());
+                        let example_name = t.example().map(std::borrow::ToOwned::to_owned);
```
Comment (dabreegster) [inline osm2lanes-web/src/control.rs:55]: Less readable IMO, but I'm not here to argue with clippy!

File: osm2lanes-web/src/draw.rs [hunk 2]
```diff
@@ -193,20 +117,113 @@ pub fn lanes<R: RenderContext>(
     Ok(())
 }
 
-pub fn draw_arrow<R: RenderContext>(
+// TODO: refactor
+#[allow(clippy::too_many_arguments)]
```
Comment (dabreegster) [inline osm2lanes-web/src/draw.rs:121]: Honestly the type system saves us here, just imagine this is Python

**Review by dabreegster — APPROVED** [review]

File: osm2lanes-web/src/control.rs (hunk 1 above)
Comment (droogmic) [reply osm2lanes-web/src/control.rs:55]: yeah, it's strange that `.to_owned` is in scope by default, but `ToOwned::to_owned` is not.

File: osm2lanes-web/src/draw.rs (hunk 2 above)
Comment (droogmic) [reply osm2lanes-web/src/draw.rs:121]: yeah,  totally workable and maintainable, just a bit of a code smell though.

**Pull request merged**